docs/contributors/release.md

```diff
@@ -1,162 +1,142 @@
-3. Extract the contents of the zip file to `trunk`.
-4. Edit `readme.txt`, replacing the changelog for the previous version with the current release's changelog.
-5. Add the changelog for the current release to `changelog.txt`.
-6. Add new files/remove deleted files from the repository:
-```bash
-# Add new files:
-svn st | grep '^\?' | awk '{print $2}' | xargs svn add
-# Delete old files:
-svn st | grep '^!' | awk '{print $2}' | xargs svn rm
-```
-7. Commit the new version to `trunk`:
-```bash
-# Replace vX.X.X-rc.1 with your version:
-svn ci -m "Committing Gutenberg version vX.X.X-rc.1"
-```
+#### Creating Release Candidate Patches (done via `git cherry-pick`)
 
-## Creating Release Candidate Patches (done via `git cherry-pick`)
+If a bug is found in a release candidate and a fix is committed to `master`, we should include that fix in a new release candidate. To do this you'll need to use `git cherry-pick` to add these changes to the milestone's release branch. This way only fixes are added to the release candidate and not all the new code that has landed on `master` since tagging:
 
-If a bug is found in a release candidate and a fix is committed to `master`, we should include that fix in a new release candidate. To do this you'll need to use `git cherry-pick`. This way only fixes are added to the release candidate and not all the new code that has landed on `master` since tagging:
-
-1. Create a pull request against master that only bumps the release candidate number in `gutenberg.php`, `package.json`, and `package-lock.json`. If your current release candidate is `v3.8.0-rc.1`, you would change the version number to `v3.8.0-rc.2`
-2. After this pull request is merged into master, note its commit SHA.
-3. Run `git fetch --tags` to load the tags.
-4. Check out the latest release candidate (for example: `git checkout v3.8.0-rc.1`).
-5. Cherry-pick fix commits (in chronological order) with `git cherry-pick [SHA]`.
-6. Cherry-pick the version bump commit you noted in step 2.
-7. Tag this release by incrementing its `rc.X` number and push it to GitHub:
-```bash
-git tag v3.8.0-rc.2
-git push origin v3.8.0-rc.2
-```
-8. Follow the steps in [tag the release](#tag-the-release), [build the plugin](#build-the-plugin), [publish the release on GitHub](#publish-the-release-on-github), and [commit to the plugin repository](#commit-to-the-plugin-repository). You can copy the existing changelog from the previous release candidate.
+1. Checkout the corresponding release branch with: `git checkout release/x.x`.
+2. Cherry-pick fix commits (in chronological order) with `git cherry-pick [SHA]`.
+3. Create [a commit like this](https://github.com/WordPress/gutenberg/pull/13125/commits/13fa651dadc2472abb9b95f80db9d5f23e63ae9c), bumping the version number in `gutenberg.php`, `package.json`, and `package-lock.json` to `x.x.0-rc.2`.
+4. Create a Pull Request from the release branch into `master` using the changelog as a description and ensure the tests pass properly.
```

Comment:
>using the changelog as a description

Is this the changelog of the main release? Or just the changes of the fixed items?

Prior art: #10893